sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -121,6 +123,11 @@ private[hive] object SparkSQLCLIDriver extends Logging {
       }
     }
 
+    Option(new HiveDelegationTokenProvider)
```

Comment:
What are you trying to achieve? `new` either returns something or throws an exception, so either you get `Some(foo)` here or an exception.

Reply:
it will get a Some and filtered if token unneeded